philippa-choong/week4/mon/calc.rb

```diff
@@ -0,0 +1,138 @@
+# Calculator
+# Explanation
+
+#     You will be building a calculator which can perform multiple arithmetic operations. Your program should allow the user to choose which operation to perform, enter the values on which the operation is performed, and output the result.
+
+
+
+require 'colorize'
+
+
+
+
+#----------Quit Calculator----------
+
+
+#--------------Arithmetic---------------
+#  Phase 1
+
+#     Calculator functionality
+#     Calculator should be able to do basic arithmetic (+,-, *, /)
+# Phase 2
+
+#     Advanced Calculator functionality
+#     Calculator should be able to do more complex arithmetic (exponents, square roots) - STFW
+
+
+
+
+#------------mortgage--------
+def mortgage
+    puts "please get a legit mortgage broker".magenta
+end
+
+
+#-------BMI---------------
+# BMI = kg/m2
+
+
+def bmi 
+    def enter_weight
+        print "enter weight (in kg): "
+        gets.to_i.to_f
+    end
+
+    def enter_height
+        print "enter height (in meters): "
+        gets.to_i.to_f
+    end
+
+    puts (enter_weight/(enter_height**2)).to_s.blue
+end
+
+
+# Specification:
+
+#     A user should be given a menu of operations
+#     A user should be able to choose from the menu
+#     A user should be able to enter numbers to perform the operation on
+#     A user should be shown the result
+#     The menu should be displayed again, and process should continue until the user selects a quit option from the menu
+def menu
+    
+ 
+    def arithmetic
+    
+        def enter_value
+            print "enter number: "
+            gets.to_i.to_f
```

Comment:
`get.to_i.to_f` is redundant because you are converting to integer then converting to float.
you cant convert anything to float after you have converting into an integer. your BMI does not work if i say 1.7m